.pre-commit-config.yaml

```diff
@@ -43,12 +43,12 @@ repos:
     args: ["--toml", "pyproject.toml"]
     additional_dependencies: ["tomli"]
 
-- repo: https://github.com/PyCQA/docformatter
-  rev: v1.7.5
-  hooks:
-  - id: docformatter
-    stages: [manual]
-    args: ["--config", "pyproject.toml"]
+# - repo: https://github.com/PyCQA/docformatter
+#   rev: v1.7.5
+#   hooks:
+#   - id: docformatter
+#     stages: [manual]
+#     args: ["--config", "pyproject.toml"]
```

Comment:
This hook was giving the following error consistently and we were not using it in CI and locally therefore we have commented it.

![image](https://github.com/user-attachments/assets/f1704416-c1d3-4371-8225-ffd830c9077c)

Reply:
That means it is not getting used either locally or in CI right?

Reply:
Yes, that was the existing behavior therefore we have commented it now.

Reply:
Shall we remove it then?

Reply:
We have kept the job below it therefore applied the same for this.

Reply:
Yeah, let's it as commented. Maybe we'll try to fix it in a future version of docformatter.